translations/translated/updateqm.pri

```diff
@@ -64,7 +65,15 @@ isEmpty(LUA_SEARCH_OUT) {
         LUA_COMMAND = "lua51"
     }
 } else {
-    LUA_COMMAND = "lua5.1"
+    win32 {
+        # The '.' in the file name confuses the QMake system() utility or the
+        # shell used in the Windows case as it is considered to be an extension
+        # unless we put in the actual 'exe' extension as well:
+        LUA_COMMAND = "lua5.1.exe"
```

Comment:
I'm not sure what are we supposed to do with the comment this information provides?

Reply:
I should have said `.1` there admittedly - and it was the explanation why we do have `LUA_COMMAND = "lua5.1.exe"` rather than `LUA_COMMAND = "lua5.1"` - as I say, it caught me out with MSYS2 as that IS the executable that is used to start a lua interpreter on the command line.

Reply:
Aha! That makes sense.

Reply:
Curiously `lua5.1` is also the one that gets used by the Travis CI used on macOs as well...